src/lsp-server.ts

```diff
@@ -367,6 +367,10 @@ export class LspServer {
     }
 
     private runCtagsOnSingleFile(rootPath: string, filePath: string) {
+        if (filePath.trim().startsWith('-')) {
+            this.logger.error(`Invalid file name: ${filePath}`);
+            return;
+        }
```

Comment:
This seems to suffice. Tried with tabs and null bytes to trip up the args parser, but seems fine.